downstream/assemblies/platform/assembly-controller-execution-environments.adoc

```diff
@@ -0,0 +1,9 @@
+[id="assembly-controller-execution-environments"]
+
+= Execution environments
+
+Unlike legacy virtual environments, execution environments are container images that make it possible to incorporate system-level dependencies and collection-based content. 
```

Comment:
There's an attribute for execution environments if you want to add that {ExecEnvShort}.